server/src/main/java/io/littlehorse/common/model/getable/core/usertaskrun/UserTaskRunModel.java

```diff
@@ -337,6 +339,14 @@ private void scheduleTaskReassign(UTActionTriggerModel action) {
         LocalDateTime localDateTime = LocalDateTime.now().plusSeconds(delayInSeconds);
         Date maturationTime =
                 Date.from(localDateTime.atZone(ZoneId.systemDefault()).toInstant());
+        ReassignUserTask command = buildReassignUserTaskCommandFrom(action);
+        if (command != null) {
```

Comment:
Nit pick: It's not clear when `command` would be null. Perhaps we could put the test for whether it would be null in this function, and then guarantee that `buildReassignUserTaskCommandFrom()` always returns a value?